react/ProductSummary.js

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import Button from '@vtex/styleguide/lib/Button'
 import { FormattedMessage } from 'react-intl'
 
+import Card from '@vtex/styleguide/lib/Card'
```

Comment:
We decided don't use the Card component.

Reply:
We decided to show something when hover. That was done